toolchain/check/facet_type.cpp

```diff
@@ -281,6 +282,120 @@ auto IsPeriodSelf(Context& context, SemIR::InstId inst_id) -> bool {
   return false;
 }
 
+// If `inst_id` is an `ImplWitnessAccess` into `.Self`, return it. Otherwise,
```

Comment:
```suggestion
// If `inst_id` is the ID of an `ImplWitnessAccess` into `.Self`, return it. Otherwise,
```

(As written it sounds like the function returns the argument, which would be a little surprising.)

Reply:
Done